recipes/munkres/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "munkres" %}
+{% set version = "1.0.12" %}
+{% set sha256 = "70b3b32b4fed3b354e5c42e4d1273880a33a13ab8c108a4247140eb661767a0b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bmc/{{ name }}/archive/release-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  source_files:
+    - test/*
+  requires:
+    - nose
+  commands:
+    - nosetests
+
+about:
+  home: https://github.com/bmc/munkres
+  license: Apache v2
```

Comment:
`license: Apache-2.0`

Reply:
One of these days we'll come up a list of these things (for at least the common ones).

Reply:
I just go off of:

https://opensource.org/licenses/alphabetical

as mentioned in the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L49-L51